plugin/cache/README.md

```diff
@@ -69,6 +70,10 @@ cache [TTL] [ZONES...] {
   greater than 5 minutes.
 * `disable`  disable the success or denial cache for the listed **ZONES**.  If no **ZONES** are given, the specified
   cache will be disabled for all zones.
+* `keepttl` enforce the cache plugin to not reduce the TTL he got from its backend. This can be useful
```

Comment:
```suggestion
* `keepttl`  do not age the TTL when serving responses from cache. The entry will still be removed from cache when the TTL expires as normal, but until it expires responses will include the original TTL instead of the remaining TTL. This can be useful 
```